### ISSUE TITLE
After update to latest BB10 OS HG10 no longer gets added to hub and cannot load conversations.

### ISSUE THREAD

**Issue by rdvl97** [amonchakai/Hg10#115]
Oddly enough, google drive integration still works flawlessly.  Although, I kinda prefer to use this for hangouts conversations. :/

**Comment by amonchakai** [comment]
Yeah, sorry about that. I know I have an issue with it, but it is hard to test as it only appends after OS update. 
To solve it, remove Hg10, get account clearer on the store:
https://appworld.blackberry.com/webstore/content/59956050
delete the Hub entry for Hg10, and reinstall it.

It should solve it

**Comment by rdvl97** [comment]
Thanks man! it works now! :D  
Perhaps you could have it check for any previous entries in the account registry upon first startup and ask the user if they wish to remove it before creating a new entry?

**Comment by amonchakai** [comment]
It is what it is trying to do, but in some case, the api to list the accounts freeze the headless. 
That is why it is waiting for the list of accounts and do not respond to login request...

**Comment by rdvl97** [comment]
Thats weird, I can't imagine why it would be doing that.  Then again, my field of expertise isn't in lower level languages and api's (I'm more of a python kinda guy). :P
